Jenkinsfile

```diff
@@ -1,5 +1,5 @@
-def targetBranch = env.getEnvironment().get('CHANGE_TARGET', env.BRANCH_NAME)
-
-library "kubic-jenkins-library@${targetBranch}"
+library identifier: "kubic-jenkins-library@master", retriever: modernSCM(
```

Comment:
We should probably keep `targetBranch` variable and use:

`
library identifier: "kubic-jenkins-library@${targetBranch}"
`

?